packages/tailwindcss-config/index.js

```diff
@@ -96,6 +96,7 @@ module.exports = {
       'gray-8': '#a2a2a2',
       'gray-9': '#c4c4c4',
       // 'gray-10': '#d9d9d9',
+      'gray-11': '#292D36',
```

Comment:
I just said `gray-11` as an example :D The naming itself should come from Gilad but since it's a really dark color, I'd say it will be something like `gray-1.5`.

I am gonna approve the PR for testing but ask Gilad what should be the name and change it later pls.

Reply:
verified with gilad, gray-90 will be used